tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf

```diff
@@ -0,0 +1,129 @@
+#
+# Pipeline definitions
+#
+# PCM0 ---> gain ----> Mixin ----> Mixout ----> gain ----> EQ IIR ----> DTS ----> SSP0
+# PCM1 ---> gain ----> Mixin ----> Mixout ----> gain ----> EQ IIR ----> DTS ----> SSP1
+
+# Pipeline ID:1 PCM ID: 0
+Object.Pipeline {
+	# playback pipelines
+	host-copier-gain-mixin-playback [
+		{
+			index	1
```

Comment:
@jsarha @bardliao @plbossart do we still need index, IIRC we were (planning?) removing index from topology to be automatic.

Reply:
@lgirdwood yes it is still needed. There's no plan to make it automatic right now because widgets are identified with the index for top-level routes.